### PR TITLE
Fix Reps devDependencies

### PR DESCRIPTION
**Pull request by nchevobbe**

### CODE REVIEW

File: packages/devtools-reps/package.json [hunk 1]
```diff
@@ -44,7 +44,9 @@
     "jest-cli": "^20.0.4",
     "jest-flow-transform": "^1.0.1",
     "react-addons-test-utils": "=15.3.2",
-    "redux-logger": "=3.0.6"
+    "react-redux": "4.4.5",
+    "redux-logger": "=3.0.6",
+    "redux": "3.5.2"
```
Comment (jasonLaster) [inline packages/devtools-reps/package.json]: Shouldn't redux come from the root package?

File: packages/devtools-reps/package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
   },
   "devDependencies": {
     "devtools-config": "^0.0.15",
-    "devtools-launchpad": "^0.0.87",
+    "devtools-launchpad": "^0.0.89",
```
Comment (jasonLaster) [inline packages/devtools-reps/package.json]: Nice catch.

File: packages/devtools-reps/package.json (hunk 1 above)
Comment (nchevobbe) [inline packages/devtools-reps/package.json]: You mean I should add the redux entries directly in `/package.json` ?

**Comment by jasonLaster** [comment]
I think it might be there. Will be at my computer soon

**Pull request closed without merging**